.eslintignore

```diff
@@ -1,6 +1,5 @@
 lib/
 dist/
 docs/
-examples/
```

Comment:
I think I'll just strip this commit and do separately (or make an issue for somebody new to contribute). That example is actually doing more and running eslint in webpack. I don't see a good reason to do that with top-level linting but it's already the case so meh.